pipeline_dp/accumulator.py

```diff
@@ -174,6 +185,47 @@ def compute_metrics(self) -> float:
 
 
 @dataclass
+class VectorSummationParams:
+    noise_kind: NoiseKind
+    max_norm: float
+
+
+class VectorSummationAccumulator(Accumulator):
+    _vec_sum: np.ndarray
+
+    def __init__(self, params: VectorSummationParams,
+                 values: Iterable[Tuple[float]]) -> None:
+        self.sum_params = params
+        values = iter(values)
+        try:
+            self._vec_sum = np.array(next(values))
+            for val in values:
```

Comment:
values is already iterable, why not simply
`for val in values: `
w/o processing of StopIteration ?

We can assume that values is not empty (this object is created after GroupBy by key for each key which is present)

Reply:
In the tests we have empty accumulators like `XAccumulator(None, []).add_value(x)`.  
IMHO this is an edge case that must be treated in an expected manner. For example, a user that wrote some custom code for joining two tables and then grouped by some multi-key might make a mistake in implementation, and they should be able to get an indication of what went wrong.  
We can either leave the aggregated value as `None` or throw an error. @dvadym up to you :)